src/ansible_navigator/actions/builder.py

```diff
@@ -0,0 +1,95 @@
+"""Run the builder subcommand."""
```

Comment:
(educational, important) Modules are not actions. Just importing them does not create side effects of performing actions (well, at least it shouldn't). When you say that something does things, it usually happens in the context of invoking functions or methods — for those, do use verbs/imperative mood. Module docstrings, OTOH, should just describe what the module contains. This one contains a class definition. By importing the module the class becomes defined/available in runtime but this is not a request to execute things. Well, looking at the source, there's one action, in fact — self-registration within the actions subsystem. Although this may be a controversial practice by itself, it seems to be a default approach in this project. I guess it may be worth mentioning too.
```suggestion
"""Builder subcommand implementation.

Importing this module registers this subcommand in the external
global subcommand registry.
"""
```

Reply:
Fixed